js/modules.js

```diff
@@ -537,24 +537,30 @@ new Module("Reconnect")
 new Module("Functions")
 .setAuthor("Gorlem")
 .setDescription("Adds functions to macromod.")
-.addAction("function <fucntionname>([...parameters])","Starts a new function block")
+.addAction("function <functionname>([...parameters])","Starts a new function block")
 .addAction("endfunction","Ends a function block")
 .addAction("[value =] call(<functionname>,[...parameters])","Calls the function")
 .addAction("[value =] <functionname>([...parameters])","Alias to [value =] call(<functionname>,[...parameters])")
-.addAction("return <value>","Sets the return value of the function and stops it")
+.addAction("return(<value>,[value 2,value 3...]","Sets the return value of the function and stops it. Use more than one parameter or return an array variable to return more than one information")
```

Comment:
Better:
Return value can be a constant, a variable, a comma seperated list or an array